src/azure-cli-core/MANIFEST.in

```diff
@@ -2,3 +2,4 @@ include LICENSE.txt
 include *.rst
 include azure/__init__.py
 include azure/cli/__init__.py
+graft azure/cli/core/tests
```

Comment:
`azure/cli/core/tests` is not necessary when you are using the `azure-cli-core` package. Homebrew installs from source distributions. Adding those tests will increase package size.

Could you provide more justification for why you need tests to be [included in the source distributions](https://packaging.python.org/en/latest/guides/using-manifest-in/)?

Reply:
When using the sdist as a source for packaging in Linux distributions, it would be nice to have the tests included there. The `azure-cli` repo source is fairly large now, especially with the vcr test cassettes included.

Reply:
Hmm, okay. I'll go about this another route then. 🤔